lib/utils/database.dart

```diff
@@ -6,7 +6,7 @@ import 'package:path/path.dart';
 import 'package:sqflite/sqflite.dart';
 
 final Map<int, List<String>> migrations = {
-  0: [
+  1: [
```

Comment:
Why did this change?

Reply:
Database cant be version 0. Not too sure why but it crashes if it is